packages/icons/src/van/VanIcon.scss

```
@@ -11,30 +11,30 @@
   src:
     url('//at.alicdn.com/t/c/font_2553510_ciljc7axaw7.woff2?t=1705587463221')
     format('woff2'),
     url('//at.alicdn.com/t/c/font_2553510_ciljc7axaw7.woff?t=1705587463221')
     format('woff');
+  font-display: auto;
 }
 
 .van-icon {
   position: relative;
   display: inline-block;
+  font-family: 'vant-icon', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "vant-icon" (font-family-name-quotes)

(font-family-name-quotes)